src/main/java/com/monumental/controllers/MonumentController.java

```diff
@@ -72,12 +98,22 @@ public Monument getMonument(@PathVariable("id") Integer id,
     }
 
     /**
-     * Get all of the Monuments
+     * Get all of the Monuments and is active or inactive depending on onlyActive
+     * @param onlyActive - If true, only active monuments will be returned. If false, monuments
+     *                 will be returned regardless of whether they're active or inactive.
+     *                 If this is false then the user must be a partner or researcher to view
```

Comment:
Can you correct this to be `If this is false then the user must be a partner or above to view`?